src/buildHTML.js

```diff
@@ -1491,6 +1491,22 @@ groupTypes.mclass = function(group, options) {
     return makeSpan([group.value.mclass], elements, options);
 };
 
+groupTypes.transform = function(group, options) {
+    const span = groupTypes.text(group, options);
+    if (group.value.dy) {
+        const dy = group.value.dy.value;
+        span.style.top = "" + (-dy.number) + dy.unit;
+        // Not working yet:
+        if (dy.number > 0) {
+            span.height += calculateSize(dy.number, options.style);
+        } else {
+            span.depth -= calculateSize(dy.number, options.style);
+        }
```

Comment:
Inside a `\frac` or `\sqrt`, the bounding box looks good:
![raisebox](https://cloud.githubusercontent.com/assets/16403058/26038466/ccaa188c-38bd-11e7-8299-912fc01cb6d6.PNG)

This screenshot, by the way, was done with code that was slightly tweaked:

```
groupTypes.transform = function(group, options) {
    const span = groupTypes.text(group, options);
    const dy = calculateSize(group.value.dy.value, options.style);
    if (dy > 0) {
        span.height += dy;
    } else if (dy < 0) {
        span.depth -= dy;
    }
    span.style.top = -dy + "em";
    span.style.position = "relative";
    return span;
};
```